ui/raidboss/data/07-dt/hunts/shaaloani.ts

```diff
@@ -434,6 +562,233 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    // ****** Boss Fate: Ttokrrone ****** //
+    // Casts fang/right/left/tail-ward sandspout then turns that way to cleave.
+    // This unfortunately rotates the boss if there happens to be a Tempest after...
+    {
+      id: 'Hunt Ttokrrone Sandspout',
+      type: 'StartsUsing',
+      netRegex: { id: ['91C1', '91C2', '91C3', '91C4'], source: 'Ttokrrone', capture: true },
+      durationSeconds: 4.9,
+      response: (data, matches, output) => {
+        const pattern = data.ttokSandOrbPatterns[data.ttokSandOrbOnSet];
+        let sandspoutPattern = TtokSafeSpots.Out;
```

Comment:
```suggestion
        let sandspoutPattern: number = TtokSafeSpots.Out;
```

Reply:
Yes, the first pass review I did was to address the compiler and test errors that were blocking the workflow runs from completing.  @wexxlee's suggestions are more comprehensive.